rskj-core/src/main/java/co/rsk/rpc/netty/Web3HttpServer.java

```diff
@@ -67,7 +70,7 @@ protected void initChannel(SocketChannel ch) throws Exception {
                     ChannelPipeline p = ch.pipeline();
                     p.addLast(new HttpRequestDecoder());
                     p.addLast(new HttpResponseEncoder());
-                    p.addLast(new HttpObjectAggregator(1024 * 1024 * 5));
+                    p.addLast(new HttpObjectAggregator(maxPayloadSize.intValue()));
```

Comment:
`maxPayloadSize` unboxing is not needed here.

Reply:
Done